Directory.Build.props

```
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+
+  <PropertyGroup>
+    <Copyright>Copyright &#xA9; 2000-2010 AlphaSierraPapa // Copyright &#xA9; 2019 Git Extensions</Copyright>
+    <Description>This is a GitExtensions fork of SharpDevelop .NET text editor control</Description>
+    <PackageId>ICSharpCode.TextEditor v4.0.2.6466</PackageId>
```

Version should be pulled out to its own property.

`PackageId` is not needed, especially in `Directory.Build.props`. It defaults to `AssemblyName`.

```suggestion
    <Version>4.0.2.6466</Version>
```